R/render.r

```diff
@@ -422,6 +422,11 @@ data_deps <- function(pkg, depth) {
       )
     )
   )
+
+  pkgdown_sass <- path_pkgdown("css", paste0("BS", bs_version), "pkgdown.css")
+  pkgdown_css <- sass::sass_partial(paste(read_lines(pkgdown_sass), collapse = ""), bs_theme)
+  bs_theme <- bslib::bs_add_rules(bs_theme, pkgdown_css)
+
   deps <- bslib::bs_theme_dependencies(bs_theme)
```

Comment:
For debugging it's nice to sometimes switch to

```r
deps <- bslib::bs_theme_dependencies(bs_theme, sass::sass_options_get(output_style = "expanded"))
```

to get a CSS that's readable.

Reply:
Because the minified version is smaller.

Reply:
I'm not sure if that's worth worrying about; or if we do worry about we should assume it happens at a later stage.